addons/web/static/src/views/standard_view_props.js

```diff
@@ -31,7 +31,7 @@ export const standardViewProps = {
     groupBy: { type: Array, elements: String },
     limit: { type: Number, optional: true },
     noBreadcrumbs: { type: Boolean, optional: true },
-    orderBy: { type: Array, elements: String },
+    orderBy: { type: Array, elements: Object },
```

Comment:
fun fact: this change has no effect, because there's a typo. In the props validation, you can describe the type of elements inside the array with the key **element** not **elements**. So basically for now, the type is totally ignored. We would have a traceback in debug mode if there was no typo. So you're right, it's no String but Object, but you should correct the typo as well to activate the full props validation (there other occurrences here).

Reply:
Hello @aab-odoo  :wave: ,

Thank you for having a look at it :smile: I fixed the typo in the standard view props. I also updated the JSDoc of the search model for the `orderBy` props. I can maybe be more precise and use `orderTerm` typescript like definition instead of just `Object` ? I think it can really help people figuring out what arguments should be provided to the functions.

Apart from that, do you agree with the changes ? Did I miss to update something ?

Reply:
Hello @julien-banken Thanks. Ok for the `orderTerm` typedef, you're right it's better than a simple Object. I'll let @Polymorphe57 review/r+ the rest of the PR